src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,35 @@
+package duke.command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * Represents a command from the user.
+ */
+public abstract class Command {
+
+    /**
+     * Creates a new Command.
+     */
+    public Command() {
+    }
+
+    /**
+     * Executes the given command with the specified tasks, ui and storage.
+     * @param tasks The tasks the command is executed with.
+     * @param ui The ui the command is executed with.
+     * @param storage The storage the command is executed with.
+     * @throws DukeException If there was a problem with executing the command.
+     */
```

Comment:
This comment is missing the empty line between description and parameter section as stated in the coding convention page. Happens a few times in this class, so I'll just comment once here!

Reply:
Noted!